gradoop-flink/src/main/java/org/gradoop/flink/io/impl/csv/CSVDataSource.java

```diff
@@ -55,14 +54,15 @@ public CSVDataSource(String csvPath, GradoopFlinkConfig config) {
   }
 
   /**
-   * {@inheritDoc}
-   * <p>
-   * Graph heads will be disposed at the moment. The following issue attempts to provide
-   * alternatives to keep graph heads: https://github.com/dbs-leipzig/gradoop/issues/974
+   * Will use a single graph head of the collection as final graph head for the graph.
```

Comment:
I think if you remove the @inheritDoc you either have to add @returns and @throws declaration or still let the @inheritDoc there because you extend (and don't want to overwrite) the existing doc.

Reply:
Please also fix the typo in the DataSource interface `* @return logial graph`